tests/unit/index.test.js

```diff
@@ -1,5 +1,49 @@
-import { describe, expect, it } from "vitest";
+import { describe, expect, it, vi } from "vitest";
+import { shortAddr } from '/src/shared/lib/utils';
+import { isAuthenticated } from "$shared/lib/auth"; 
+import { get } from "svelte/store";
 
+
+// Unit-tests for “wallet” variable (?)and for "isAuthenticated" function: 
+// export function isAuthenticated() {
+// 	return !isEmpty(get(wallet));
+//   }
+describe("isAuthenticated", () => {
+	it("must return 'false' for an empty wallet", () => {
+		vi.mocked(get).mockReturnValueOnce({}); // TypeError: vi.mocked(...).mockReturnValueOnce is not a function
```

Comment:
It's not expected during correct mocking to have such error `TypeError: vi.mocked(...).mockReturnValueOnce is not a function.`
The ﻿`vi.mocked` function allows TypeScript users to cast an object to a mocked type, although it does not offer any mocking functionality on its own.
Check this doc https://vitest.dev/guide/mocking.html, it shows good examples of correct mocking in vitest.